### ISSUE TITLE
 Numeric value out of range: blitz_elementqueries (PostgreSQL)

### ISSUE THREAD

**Issue by jake-pump** [putyourlightson/craft-blitz#59]
I've encountered an issue with the 'blitz_elementqueries' table. The value that gets assigned to the 'index' column sometimes exceeds the numerical range limit for field type 'integer'. I am experiencing this issue in a Postgres DB specifically. Below is an example of one such error:

```
SQLSTATE[22003]: Numeric value out of range: 7 ERROR: value "2571137399" is out of range for type integer
The SQL being executed was: SELECT "id"
FROM "blitz_elementqueries"
WHERE "index"='2571137399'
```

I am unsure why such high numeric values end up being used. However, switching the field to 'bigint' is a possible workaround.

**Comment by putyourlightson** [comment]
I hadn't realised that unsigned integers are unavailable in Postgres. Yes, using `bigint` is probably what we'll do in the next release then. Thanks for reporting this!

**Comment by putyourlightson** [comment]
FYI the `index` column is not an `id` but an index that uniquely defines the query parameters using a CRC32 hashing algorithm for quicker indexing while saving space.

**Comment by putyourlightson** [comment]
Fixed in version 1.11.2.
